src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -113,35 +132,74 @@ func main() {
 @end
 
 @private initialize(method)
-  @join param : context.getFlatMethodParams(method) on BREAK.add(BREAK)
-    @let signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
-        paramField = context.getField(signatureType, param), \
-        paramValue = context.typeDefaultValue(signatureType, paramField), \
-        paramDescription = context.getApiaryConfig.getDescription(signatureType.getName, param)
-      {@description(paramDescription)}
-      {@unclashVar(param)} := {@paramValue} // {@TODO()} Update placeholder value.
+  @let methodName = context.lowerCamelToUpperCamel(context.getMethodName(method)), \
+       signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
+       params = context.getFlatMethodParams(method)
+    @if params
+      @join param : params
+        @let paramField = context.getField(signatureType, param), \
+             paramValue = context.typeDefaultValue(signatureType, paramField), \
+             paramDescription = context.getDescription(signatureType.getName, param)
+          {@description(paramDescription)}
+          {@unclashVar(param)} := {@paramValue} // {@TODO()} Update placeholder value(s).
```

Comment:
I mean... we could... but is so much work for a simple thing worth it? I think `value(s).` is sufficient